src/openrct2/localisation/string_ids.h

```diff
@@ -3643,9 +3643,16 @@ enum {
 	STR_MAP_ELEMENT_LIMIT_REACHED = 5992,
 	STR_TILE_INSPECTOR_COPY_TIP = 5993,
 	STR_TILE_INSPECTOR_PASTE_TIP = 5994,
-
 	STR_BOOSTER = 5995,
 	STR_RIDE_CONSTRUCTION_BOOSTER_SPEED = 5996,
+	STR_PATH_SCENERY = 5997,
```

Comment:
Add blank line, plus convert everything to `FOOTPATH_SCENERY` just to be consistent (Theme entry has one `O` too many)